packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -293,7 +293,7 @@ export class List extends BaseComponent<IListProps, IListState> {
     let divProps = getNativeProps(this.props, divProperties);
 
     // assign list if no role
-    role = role || 'list';
+    role = (typeof role === 'undefined') ? 'list' : role;
```

Comment:
That's what I said :) Seems fragile to rely on null being of type object to allow it to skip that test.